pkgs/by-name/je/jellyfin-tui/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "v1.0.5";
+
+  src = fetchFromGitHub {
```

Comment:
Can we add an automatic package update script using `nixpkgs-update`? See [documentation](https://nix-community.github.io/nixpkgs-update/).
```nix
passthru.updateScript = nix-update-script { }; 
```